### ISSUE TITLE
bower install throws :   bower EINVRES       Request to https://bower.herokuapp.com/packages/angular-mocks failed with 410

### ISSUE THREAD

**Issue by jpca999** [meanjs/mean#1899]
![image](https://user-images.githubusercontent.com/1026325/31440996-d46b56ea-ae5f-11e7-8058-4d4ca57a112b.png)

**Comment by simison** [comment]
Must've been temporary hiccup as now it seems to work just fine:
```
> bower install --allow-root && bower prune --allow-root

bower owasp-password-strength-test#~1.3.0       not-cached https://github.com/viaforensics/owasp-password-strength-test.git#~1.3.0
bower owasp-password-strength-test#~1.3.0          resolve https://github.com/viaforensics/owasp-password-strength-test.git#~1.3.0
bower angular-ui-notification#~0.2.0            not-cached https://github.com/alexcrack/angular-ui-notification.git#~0.2.0
bower angular-ui-notification#~0.2.0               resolve https://github.com/alexcrack/angular-ui-notification.git#~0.2.0
bower angular-bootstrap#~1.2.1                  not-cached https://github.com/angular-ui/bootstrap-bower.git#~1.2.1
bower angular-bootstrap#~1.2.1                     resolve https://github.com/angular-ui/bootstrap-bower.git#~1.2.1
bower angular-messages#~1.5.0                   not-cached https://github.com/angular/bower-angular-messages.git#~1.5.0
bower angular-messages#~1.5.0                      resolve https://github.com/angular/bower-angular-messages.git#~1.5.0
bower ng-file-upload#~12.1.0                    not-cached https://github.com/danialfarid/angular-file-upload-bower.git#~12.1.0
bower ng-file-upload#~12.1.0                       resolve https://github.com/danialfarid/angular-file-upload-bower.git#~12.1.0
bower angular-ui-router#~0.2.18                 not-cached https://github.com/angular-ui/angular-ui-router-bower.git#~0.2.18
bower angular-ui-router#~0.2.18                    resolve https://github.com/angular-ui/angular-ui-router-bower.git#~0.2.18
bower angular#~1.5.0                                cached https://github.com/angular/bower-angular.git#1.5.11
bower angular#~1.5.0                              validate 1.5.11 against https://github.com/angular/bower-angular.git#~1.5.0
bower angular-animate#~1.5.0                        cached https://github.com/angular/bower-angular-animate.git#1.5.11
bower angular-animate#~1.5.0                      validate 1.5.11 against https://github.com/angular/bower-angular-animate.git#~1.5.0
bower angular-resource#~1.5.0                       cached https://github.com/angular/bower-angular-resource.git#1.5.11
bower angular-resource#~1.5.0                     validate 1.5.11 against https://github.com/angular/bower-angular-resource.git#~1.5.0
bower bootstrap#~3.3.6                              cached https://github.com/twbs/bootstrap.git#3.3.7
bower bootstrap#~3.3.6                            validate 3.3.7 against https://github.com/twbs/bootstrap.git#~3.3.6
bower angular-mocks#~1.5.0                          cached https://github.com/angular/bower-angular-mocks.git#1.5.11
bower angular-mocks#~1.5.0                        validate 1.5.11 against https://github.com/angular/bower-angular-mocks.git#~1.5.0
bower angular-ui-router#~0.2.18                   download https://github.com/angular-ui/angular-ui-router-bower/archive/0.2.18.tar.gz
bower owasp-password-strength-test#~1.3.0         download https://github.com/viaforensics/owasp-password-strength-test/archive/1.3.0.tar.gz
bower ng-file-upload#~12.1.0                      download https://github.com/danialfarid/angular-file-upload-bower/archive/12.1.0.tar.gz
bower angular-ui-notification#~0.2.0              download https://github.com/alexcrack/angular-ui-notification/archive/v0.2.0.tar.gz
bower angular-bootstrap#~1.2.1                    download https://github.com/angular-ui/bootstrap-bower/archive/1.2.5.tar.gz
bower angular-ui-router#~0.2.18                    extract archive.tar.gz
bower owasp-password-strength-test#~1.3.0          extract archive.tar.gz
bower owasp-password-strength-test#~1.3.0         resolved https://github.com/viaforensics/owasp-password-strength-test.git#1.3.0
bower angular-ui-router#~0.2.18                   resolved https://github.com/angular-ui/angular-ui-router-bower.git#0.2.18
bower angular-ui-notification#~0.2.0               extract archive.tar.gz
bower angular-ui-notification#~0.2.0              mismatch Version declared in the json (0.1.0) is different than the resolved one (0.2.0)
bower angular-ui-notification#~0.2.0              resolved https://github.com/alexcrack/angular-ui-notification.git#0.2.0
bower angular-messages#~1.5.0                     download https://github.com/angular/bower-angular-messages/archive/v1.5.11.tar.gz
bower angular-bootstrap#~1.2.1                     extract archive.tar.gz
bower ng-file-upload#~12.1.0                       extract archive.tar.gz
bower angular-bootstrap#~1.2.1                    resolved https://github.com/angular-ui/bootstrap-bower.git#1.2.5
bower ng-file-upload#~12.1.0                      resolved https://github.com/danialfarid/angular-file-upload-bower.git#12.1.0
bower angular-messages#~1.5.0                      extract archive.tar.gz
bower angular-messages#~1.5.0                     resolved https://github.com/angular/bower-angular-messages.git#1.5.11
bower jquery#1.9.1 - 3                              cached https://github.com/jquery/jquery-dist.git#3.2.1
bower jquery#1.9.1 - 3                            validate 3.2.1 against https://github.com/jquery/jquery-dist.git#1.9.1 - 3
bower angular#~1.5.0                               install angular#1.5.11
bower owasp-password-strength-test#~1.3.0          install owasp-password-strength-test#1.3.0
bower angular-ui-router#~0.2.18                    install angular-ui-router#0.2.18
bower angular-ui-notification#~0.2.0               install angular-ui-notification#0.2.0
bower angular-animate#~1.5.0                       install angular-animate#1.5.11
bower angular-resource#~1.5.0                      install angular-resource#1.5.11
bower angular-bootstrap#~1.2.1                     install angular-bootstrap#1.2.5
bower ng-file-upload#~12.1.0                       install ng-file-upload#12.1.0
bower angular-mocks#~1.5.0                         install angular-mocks#1.5.11
bower angular-messages#~1.5.0                      install angular-messages#1.5.11
bower bootstrap#~3.3.6                             install bootstrap#3.3.7
bower jquery#1.9.1 - 3                             install jquery#3.2.1
```

**Comment by jpca999** [comment]
Resolved
